README.md

```diff
@@ -29,6 +29,20 @@ Note that all flags can be replaced with environment variables; for instance,
 `--dry-run` could be replaced with `EXTERNAL_DNS_DRY_RUN=1`, or
 `--registry txt` could be replaced with `EXTERNAL_DNS_REGISTRY=txt`.
 
+## Deploying to a Cluster
+
+The following tutorials are provided:
+
+* [AWS](docs/tutorials/aws.md)
+* [Azure](docs/tutorials/azure.md)
+* [Cloudflare](docs/tutorials/cloudflare.md)
+* [DigitalOcean](docs/tutorials/digitalocean.md)
+* Google Container Engine
+	* [Using Google's Default Ingress Controller](docs/tutorials/gke.md)
+	* [Using the Nginx Ingress Controller](docs/tutorials/nginx-ingress.md)
+
+## Running Locally
```

Comment:
@pluttrell Nevermind, the links are a great addition! I looked at the heading wrong.